### DISCUSSION TITLE
Options parameter not working

### DISCUSSION THREAD

**Discussion by mimimalisma** [702573N/Obsidian-Tasks-Calendar#60] [Q&A]
Hey there :) I'm new to Obsidian and may just doing something entirely wrong. I'm trying to set up the calendar, but no matter what I try, options won't work for any command

Here is my code:

````dataviewjs
await dv.view("tasksCalendar", {pages: "", view: "month", firstDayOfWeek: "1", options: "style1"})
pages: '"Termine"'
options: "noIcons"
````

I'd be very thankful for any help :)
And thanks a lot for providing this awesome template!

**Comment by reorganice** [comment]
```dataviewjs
await dv.view("tasksCalendar", {pages: '"Termine"', view: "week", firstDayOfWeek: "1", options: "style1 noIcons"})
```
... at least one style has to be chosen for the weekly view
... then you just add the options without any comma into the quotation marks ""
